apps/pie-docs/src/_utilities/tokens.js

```diff
@@ -0,0 +1,140 @@
+const { objectHelpers, numberHelpers, stringHelpers } = require('./helpers');
+const normalisedPieDesignTokens = require('../_data/normaliseTokens');
+const pieTokenCategories = require('../tokenCategories.json');
+const pieTokensMetadata = require('../tokensMetadata.json');
+const tokenTypes = require('../_data/tokenTypes');
+
+/**
```

Comment:
This is not new, it's just been moved so they can be shared between `tokenTable` and `simpleTable`